docs_espressif/en/additionalfeatures/nvs-partition-editor.rst

```diff
@@ -0,0 +1,37 @@
+NVS Partition Editor
+=========================
+
+The **ESP-IDF: Open NVS Partition Editor** allows the user to create a NVS partition binary file based on key-value pairs in CSV file. The resulting binary file is compatible with NVS architecture defined in `ESP-IDF Non Volatile Storage <https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/storage/nvs_flash.html>`_.
```

Comment:
For consistency in addressing the reader, I would suggest addressing him directly and not use the `user` word here